custom-elements/custom-element-registry/per-global.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Custom Elements: CustomElementRegistry is per global</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#custom-elements-api">
+<link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
+<script src="/common/object-association.js"></script>
+
+<iframe id="blank-iframe"></iframe>
```

Comment:
It seems you can remove this. Meaning this could in fact be a .window.js though that would remove some metadata so up to you.